README.md

```diff
@@ -210,11 +210,32 @@ app.use(limitMiddleware.middleware(options, function(req, res, next) {
 }));
 ```
 
+Include ratelimit response headers:
+
+```javascript
+var options = {
+  headers: true
```

Comment:
As far as this formatting.. should it be named something more descriptive like ratelimit_headers? 

or maybe something like..
```
var options = {
    headers: { 
        ratelimit: true
    }
}
```
Just thinking if we want to give more option in the future for which headers to include, or even show headers unrelated to ratelimit this would be more forward thinking.